modules/develop/pages/data-transforms/run-transforms.adoc

```diff
@@ -0,0 +1,254 @@
+= Run Data Transforms
+:description: Learn how to build and deploy WebAssembly data transforms in Redpanda.
+:page-aliases: reference:rpk/rpk-wasm/rpk-wasm.adoc, reference:rpk/rpk-wasm.adoc, reference:rpk/rpk-wasm/rpk-wasm-deploy.adoc, reference:rpk/rpk-wasm/rpk-wasm-generate.adoc, reference:rpk/rpk-wasm/rpk-wasm-remove.adoc, data-management:data-transform.adoc, labs:data-transform/index.adoc
+:latest-data-transforms-version: 0.0.0-20230830git604fcce
+
+Data transforms let you run common data streaming tasks, like filtering, scrubbing, and transcoding, within Redpanda. For example, you may have consumers that require you to remove credit card numbers or convert JSON to Avro. Data transforms can also interact with the Redpanda Schema Registry to work with encoded data types. 
+
+Data transforms use a WebAssembly (Wasm) engine inside a Redpanda broker. A Wasm function acts on a single record in an input topic. You can develop and manage data transforms with xref:reference:rpk/rpk-transform/rpk-transform.adoc[`rpk transform`] commands.
+
+See also: xref:./how-transforms-work.adoc[] and <<Limitations>>
+
+IMPORTANT: This Beta feature is not supported for production deployments. Beta transforms may not work the same after the GA release. 
+
+== Enable data transforms
+
+. To enable data transforms, set the `data_transforms_enabled` cluster configuration flag:
++
+```bash
+rpk cluster config set data_transforms_enabled true
+```
+. Restart the broker with `rpk redpanda stop`, then `rpk redpanda start`. This property requires the cluster to be restarted in order to be applied.
+
+== Create a data transforms project
+
+. Create and initialize a data transforms project:
++
+```bash
+rpk transform init
+```
++
+A successful command generates project files in your current directory:
++
+[.no-copy]
+----
+.
+├── go.mod
+├── go.sum
+├── README.md
+├── transform.go
+└── transform.yaml
+----
++
+The `transform.go` file contains the transform logic, and the `transform.yaml` file configures the project.
```

Comment:
@rockwotj a nit but should we say 'configures the transform' in case we have something in the future that's a true project structure, or are we calling what we have now a 'project'?

Reply:
Yes, let's say " and the `transform.yaml file specifies the transform's configuration` or something of the like.